code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -692,7 +645,7 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 /// Gets the chance for latejoin injection, the dry_run argument is only used for forced injection.
 /datum/game_mode/dynamic/proc/get_injection_chance(dry_run = FALSE)
 	if(forced_injection)
-		forced_injection = !dry_run
+		forced_injection = dry_run
```

Comment:
This is the bug fix, CC @kriskog do you know why it was like this? (dry_run = TRUE when checking the admin panel)